ceph-iscsi-cli.spec

```diff
@@ -2,7 +2,7 @@ Name:		ceph-iscsi-cli
 Version:	2.8
 Release:	1%{?dist}
 Summary:	CLI configuration tool to manage multiple iSCSI gateways
-Group:		Applications/System
+Group:		System/Filesystems
```

Comment:
Nit: care to fix the odd indentation while you are here?

Reply:
Sure thing :-)